docs/advanced-topics/docker-alternatives.rst

```diff
@@ -40,10 +40,10 @@ More information about compatibility of dockerfiles with Singularity
 can be found `here <https://sylabs.io/guides/3.4/user-guide/singularity_and_docker.html#best-practices>`__.
 
 Singularity can be installed following the instructions
-`here <https://sylabs.io/guides/3.4/user-guide/quick_start.html>`__. Note that the installation is relatively complicated
```

Comment:
Does this deletion mean that sudo no longer required?

Reply:
I'm OK with this; IMHO it's deferring to whatever their  doc says.

Reply:
I removed the comment and replaced the link to their current Singularity's install docs that the user can refer to. The old link was specifically for Singularity version 3.4 which is now outdated.